test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -367,6 +367,34 @@ def test_callibration_results(self):
             counts_array[0], counts_array[1], msg="Counts different with/without fitter."
         )
 
+    @unittest.skipUnless(HAS_AER, "qiskit-aer is required for this test")
+    @unittest.skipUnless(HAS_IGNIS, "qiskit-ignis is required to run this test")
```

Comment:
We should not need ignis anymore right so this skip can be removed as ignis is now deprecated and for now the mitigation classes that are used were copied until Terra here until such time as this is all updated with the new mitigation logic.

Reply:
I removed the Ignis dependency from this new unit test method